components/constants/webui_url_constants.h

```diff
@@ -112,4 +111,10 @@ inline constexpr auto kInstallablePWAWebUIHosts =
         kAIChatUIHost,
     });
 
+inline constexpr char kEmailAliasesHost[] = "email-aliases";
+inline constexpr char kEmailAliasesSettingsURL[] =
+    "brave://settings/email-aliases";
```

Comment:
I don't know why there is a mix of `chrome` and `brave` in some of these, but the real url is always `chrome` so that's what we should have here

Reply:
fixed